libparsec/crates/platform_storage/src/native/workspace.rs

```diff
@@ -304,10 +306,9 @@ impl PlatformWorkspaceStorage {
     }
 
     /// Only used for debugging tests
-    #[allow(unused)]
-    pub async fn debug_dump(&mut self) -> anyhow::Result<String> {
+    #[cfg(test)]
```

Comment:
It's important to have `debug_dump` exposed to the other crates, as it is a very useful when debugging tests !

Reply:
I'm in phase with what you're saying, but I'm waiting for #7641 to be merged

Reply:
done